docbot/action.go

```diff
@@ -13,10 +13,7 @@ import (
 )
 
 const (
-	MessageLabelMissing = `Please provide a correct documentation label for your PR.
-Instructions see [Pulsar Documentation Label Guide](https://docs.google.com/document/d/1Qw7LHQdXWBW9t2-r-A7QdFDBwmZh6ytB4guwMoXHqc0).`
-	MessageLabelMultiple = `Please select only one documentation label for your PR.
-Instructions see [Pulsar Documentation Label Guide](https://docs.google.com/document/d/1Qw7LHQdXWBW9t2-r-A7QdFDBwmZh6ytB4guwMoXHqc0).`
+	MessageLabelMultiple = "Please select only one documentation label in your PR description."
```

Comment:
Why removes the `Pulsar Documentation Label Guide` info? This guide provides more context and useful info to users

Reply:
It looks cleaner, and I think these descriptions are sufficient.

Reply:
If we can convey the instructions in several lines, we don't have to link to an external link. And I suppose that our goal of the final state is to keep all info within the projects instead of a group of Google Docs.

Reply:
`Pulsar Documentation Label Guide` contains more info than `several lines (instructions after doc label)` here https://github.com/apache/pulsar-test-infra/pull/71#issuecomment-1247541039. Users (especially newcomers) should be navigated to an info-rich place to know more.

Reply:
I suggest we move the `Pulsar Documentation Label Guide` to the Pulsar contribution guide to minimize external links as much as possible.


When a contributor wants to contribute to the Pulsar, they should review the Pulsar contribution guide.

Reply:
> 1. The contents of `Pulsar Documentation Label Guide` will be moved to the Pulsar website, so the external links is a temporary state, do not need to worry about that (accessibility).

LGTM. I think these comments are enough for the contributor, but the doc contributor maybe need this doc, because it is detailed.

> 2. "How to make contributions to Pulsar" is a pretty big topic, which contains a lot of content. This content should be put into different guides (eg. `label guide`, `PR title naming guide`, `writing style guide`, `design guide`) rather than a single guide (eg. `Pulsar Contribution Guide`) since
> 
> 1. they have different subjects
> 2. Readers: do not have the patience to read a "long" book

I remember @tisonkun will do this thing.